doc/source/examples/pdp_regression_bike.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
<p>Btw <code>pandas</code> is currently a core dependency so no need to mention this or do the <code>!pip ...</code> call.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>